File: Model/Product.cs
using Microsoft.AspNetCore.Http;

namespace backendtask.Model
{
    public class Product
    {
        public int ProductID { get; set; }
        public string? ProductName { get; set; }
        public string? Country { get; set; }
        public string? InvoicePeriod { get; set; }
        public string? ScrapType { get; set; }
        public int ManCost { get; set; }
        public int MaterialCost { get; set; }
        public int EstimateCost { get; set; }
        public int LocalAmount { get; set; }
        public IFormFile Image { get; set; }
        public string ImageFileName { get; set; }
    }
}

File: src/app/product.ts
export interface Product {
    productID: number,
    productName: String,
    country: string,
    invoicePeriod: string,
    scrapType: string,
    manCost: number,
    materialCost: number,
    estimateCost: number|null,
    localAmount: number,
    image: File | undefined
}

export interface Country{
    country: string
    currency:number
}
